components/sections/SponsorShowcase.tsx

```diff
@@ -55,56 +63,120 @@ function SponsorShowcase(): ReactElement {
           justifyItems="center"
           alignItems="center"
         >
-          <SponsorImage
-            src={`${imgPath}/deloitte.png`}
+          <SponsorLink
```

Comment:
![image](https://user-images.githubusercontent.com/20251243/101718090-88bf8b00-3a6e-11eb-840c-ed064497b88d.png)

SponsorLink takes up the full grid width (see highlighted portion of above screenshot) instead of just the image. I'm thinking the widths set on the image should be moved out to a parent Box (`div`) container.

I can work on this part if you'd like!